lib/locket/client.rb

```diff
@@ -52,7 +54,7 @@ def lock_acquired?
 
     attr_reader :service, :lock_acquired, :key, :owner
```

Comment:
`key` and `owner` are no longer instance fields...

Reply:
True dat. I'll update this.